assets/timber.scss.liquid

```diff
@@ -113,6 +113,9 @@ $colorNavText: #333;
 $colorFooterBg: {{ settings.color_footer_bg }};
 $colorFooterText: {{ settings.color_footer_text }};
 
+// Logo max width
+$logoMaxWidth: {{ settings.logo_max_width | default: '450' }}px;
```

Comment:
I personally do this in the theme, so even for people including the px, it does not break:

{{ settings.logo_max_width | default: '450' | remove: 'px' }}px;